sdk/textanalytics/azure-ai-textanalytics/tests/test_recognize_linked_entities.py

```diff
@@ -586,3 +586,19 @@ def test_string_index_type_not_fail_v3(self, client):
         # make sure that the addition of the string_index_type kwarg for v3.1-preview.1 doesn't
         # cause v3.0 calls to fail
         client.recognize_linked_entities(["please don't fail"])
+
+    # currently only have this as playback since the dev endpoint is unreliable
+    @pytest.mark.playback_test_only
+    @GlobalTextAnalyticsAccountPreparer()
+    @TextAnalyticsClientPreparer(client_kwargs={
+        "api_version": TextAnalyticsApiVersion.V3_1_PREVIEW_2,
+        "text_analytics_account_key": os.environ.get('AZURE_TEXT_ANALYTICS_KEY'),
+        "text_analytics_account": "https://cognitiveusw2dev.azure-api.net/"
+    })
+    def test_bing_id(self, client):
```

Comment:
do u know if the bingId property will always be populated?
worth adding a test checking that the property has content on it

Reply:
It should be populated by the server for v3.1-preview.2 and up (it's not shown as an optional parameter). I'm confused by "worth adding a test checking that the property has content on it", I believe that's what I'm checking in this test

Reply:
It might be me and not knowing ton of Python, but `assert entity` doesn't seem to be checking that bing_id  has something on it. like it is not None or null, or empty

Reply:
assert checks that it's not `None` and not and empty string